shared/utils/enable-derivatives.py

```diff
@@ -208,13 +208,16 @@ def main():
     parser.add_option("--enable-sl", dest="sl", default=False,
                       action="store_true", help="Enable Scientific Linux")
     parser.add_option("-i", "--input", dest="input_content", default=False,
-                      action="store", help="INPUT can be XCCDF or Source DataStream")
+                      action="store",
+                      help="INPUT can be XCCDF or Source DataStream")
     parser.add_option("-o", "--output", dest="output", default=False,
                       action="store", help="XML Tree content")
     (options, args) = parser.parse_args()
 
     if options.centos and options.sl:
-        print "Cannot enable two derivative OS(s) at the same time"
```

Comment:
FYI, you could have used `print("...", file=sys.stderr)`

Reply:
good point, do you want me to change that? we use sys.stderr.write all over the place in SSG, if one approach is preferable we should switch everywhere...

Reply:
As you wish, I am fine either way. However, I have noticed that we don't use the "future imports", which is probably wrong - that way, print usage across Python2 and Python3 may be inconsistent.
Therefore, I suggest putting `from __future__ import print_function` at the top of modules that use `print`. This is supported from Python 2.6 and it will ensure that e.g. `print(1, 2)` will behave the same and also ensure the `file` kwarg availability.